editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/propertysheet/PropertiesUtils.java

```diff
@@ -311,7 +311,8 @@ public static void updateURIParams(Endpoint selectedEP, UriParameter p, Object v
     			}
     			if (val.trim().length()<1) val = pparam.getDefaultValue();
     			if (val != null && val.startsWith("/") && !CamelComponentUtils.isFileProperty(pparam)) val = val.substring(1);
-    			if (val != null) syntax = syntax.replace(pparam.getName(), val);
+    			// Workaround for FUSETOOLS-1369
+    			if (val != null) syntax = replace(syntax, pparam.getName(), val, syntax.indexOf(':'));
```

Comment:
It is not said, that the separator is always the : char. Sometimes there are even multiple different separators used in a single endpoint uri (host:port/path for instance). So your fix will not work in a generic way. I will take a closer look at your commit next week but from first sight I'd say that fix doesn't take care off all the possible edge cases.